### ISSUE TITLE
Phaser(s) interactions

### ISSUE THREAD

**Issue by sebastien-bouffier** [antoniograzioli/Autodafe#7]
Hi Antonio, I noticed that when I create a new patch with just one VCO going into your Phaser module everything is right but when I add a second one without any new virtual patch cable, controls of the second one affect the first one which is running. It sounds like both instances share the same set of internal parameters. 
Best regards.

**Comment by antoniograzioli** [comment]
yes, I think I have already managed to solve this, but maybe it's not in
the sources. I 'll double check


2017-10-28 16:41 GMT+02:00 sebastien-bouffier <notifications@github.com>:
